gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,4 +1,3 @@
-#Fri Feb 09 18:53:31 PST 2018

```

Comment:
@dhaval24 The gradle related files don't need to be updated. (gradle-wrapper.properties, gradle-wrapper.jar, gradlew)